packages/core/contracts/financial-templates/common/financial-product-libraries/long-short-pair-libraries/SmaugBondLongShortPairFinancialProductLibrary.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/utils/math/SignedSafeMath.sol";
+import "@openzeppelin/contracts/utils/math/Math.sol";
+
+import "./LongShortPairFinancialProductLibrary.sol";
+import "../../../../common/implementation/Lockable.sol";
+
+/**
+ * @title Smaug Bond Long Short Pair Financial Product Library
```

Comment:
Can we name this something less specific and more descriptive than Smaug Bond? Maybe `YieldDollarLongShortPairFinancialProductLibrary`?

Reply:
Sure, I'll go with `YieldDollar` for now unless we have something better, since the embedded put option makes it not quite a yield dollar past the low range.

Reply:
Actually, on second thought, I'll do `YieldDollarMinusPut` to be totally explicit. Can do a global find and replace if we come up with something less clunky.

Reply:
Sounds good